### ISSUE TITLE
Should preserve order of items in an album for local album folders

### ISSUE THREAD

**Issue by gilesknap** [gilesknap/gphotos-sync#57]
Need to record order items come out into the DB.

**Comment by gilesknap** [comment]
Also - need to investigate why "Y2000 M05 D06 Canada - DSCF0051.AVI" and other AVIs have the wrong Create Date in the DB (this is the reason that order of items in albums changed and that I noticed this issue - i had addded ORDER BY Date in LocalData.get_album_files() and all the AVIs ended up at the end of the album '0511 Trip to Banff and Improvement District No. 9' )

**Comment by gilesknap** [comment]
The date anomoly in AVI files is because they have incorrect metadata, see example below. Conversely the Google Photos metadata is somehow correct. When I do a local folder comparison pass I extract the create date using ffprobe and write it into the DB, thus trashing the order. The actual metadata comparison is a good way to identify if two files are a copy of the original so probably want to keep this ffprobe.

Solutions?
- dont use ffprobe and rely on Google meta data only.
- do ffprobe but have a separte column in the DB for these dates, only used for comparison
- comparison is a relativley rare (expensive) operation, could simply always re-run a normal scan with --flush-index afterward (which is what I'll do right now while I think about this)

```
$ ffprobe -v quiet -print_format json -show_entries stream=index,codec_type:stream_tags=creation_time:format_tags=creation_time '/media/Data/GilesPhotos/photos/2000/05/Y2000 M05 D06 Canada - DSCF0051.AVI'
{
    "programs": [

    ],
    "streams": [
        {
            "index": 0,
            "codec_type": "video",
            "tags": {

            }
        },
        {
            "index": 1,
            "codec_type": "audio",
            "tags": {
                "creation_time": "2015-11-05T02:03:03.000000Z"
            }
        }
    ],
    "format": {
        "tags": {
            "creation_time": "2015-11-05T02:03:03.000000Z"
        }
    }
}

```

**Comment by gilesknap** [comment]
The wrong metadata issue is separate from album ordering, splitting into a new ticket.

**Comment by gilesknap** [comment]
Top solution for this is to record position in an album in the albumFiles table. Simples.

**Comment by gilesknap** [comment]
There appears to be a solution to this here https://github.com/plevold/gphotos-sync/commit/22f74dfc9f29a5015beb35f8a5660c2a6bf2ce5d

**Comment by gilesknap** [comment]
this has been fixed (currently in master) by @plevold. Thanks!

**Comment by gilesknap** [comment]
note I need to fix #148 before releasing this

**Comment by gilesknap** [comment]
#148 is now fixed - this will go in next release.
